projects/plugins/jetpack/extensions/blocks/ai-assistant/editor.scss

```diff
@@ -30,14 +30,26 @@
 	color: var( --wp--preset--color--black );
 }
 
-.jetpack-ai-assistant__loading-spinner {
-	margin: auto 0;
-	padding-left: 4px;
+.jetpack-ai-assistant__input-icon-wrapper {
+	flex-shrink: 0;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	padding: 0 4px 0 8px;
+	
+	.jetpack-ai-assistant__input-icon {
+		// Brand green regardless of theme
+		color: #069E08; // jp-green-40
```

Comment:
Do you think we can use the CSS var here?

Reply:
We are using the CSS var to define the color in the multiple block edition panel